test/wh_test_crypto.c

```diff
@@ -273,77 +276,28 @@ int whTest_CryptoServerConfig(whServerConfig* config)
 {
     whServerContext server[1] = {0};
     int ret = 0;
-    int i;
+    int am_connected = WH_COMM_CONNECTED;
 
     if (config == NULL) {
         return WH_ERROR_BADARGS;
     }
 
     WH_TEST_RETURN_ON_FAIL(wh_Server_Init(server, config));
 
-    /* handle client rng */
-    do {
+    for (   wh_Server_SetConnected(server, am_connected);
+            am_connected == WH_COMM_CONNECTED;
+            wh_Server_GetConnected(server, &am_connected) ){
```

Comment:
this is fun

Reply:
Prolly too fancy.  I'll switch it to a while structure. Fixed in next push